src/core/utils.js

```diff
@@ -7,6 +7,15 @@ import { hyperHTML } from "./import-maps.js";
 import { pub } from "./pubsubhub.js";
 export const name = "core/utils";
 
+/**
+ * Checks whether a level is valid. Levels should be integers >= 0.
+ *
+ * @param {*} level
+ */
+export function validLevel(level) {
```

Comment:
We probably only want to export a util function if this is used in multiple places.

Reply:
it is currently used in `title.js` for the title and `header.js` for the shortName